### PR TITLE
Added copying multiple tickets answers enhancement (#149)

### PR DESCRIPTION
**Pull request by chrisdamba**
Enhancement for copying answers into multiple tickets with a checkbox state as UI

### CODE REVIEW

File: src/pretix/presale/templates/pretixpresale/event/checkout_questions.html [hunk 1]
```diff
@@ -41,6 +41,11 @@ <h4 class="panel-title">
                     </div>
                 </div>
             {% endif %}
+            <div id="cp_copy" class="panel panel-default">
+                <div class="panel-body">
+                    <input type="checkbox" name="copy" id="copy_answers">  Copy Answers
```
Comment (raphaelm) [inline src/pretix/presale/templates/pretixpresale/event/checkout_questions.html]: We need to have this translatable, you can use something like ``{% trans "Copy answers" %}`` for this.

File: src/static/pretixpresale/js/ui/main.js [hunk 2]
```diff
@@ -22,8 +22,67 @@ $(function () {
...
+        }
+    })
 });
 
+function display_copy() {
+    var elements = $("input").filter(function(){
+        return this.id.match(/^id_(\d+)/);
+    });    
+    var shouldElementBeVisible = (elements.length > 2);    
+    $("#cp_copy").toggle(!!shouldElementBeVisible);
```
Comment (raphaelm) [inline src/static/pretixpresale/js/ui/main.js]: This is already boolean, I believe there is no reason for the `!!`.

**Comment by chrisdamba** [comment]
Thanks @raphaelm for flagging the above. Of course I'm happy to revise and implement. On the UI, I felt a checkbox is a better control than a button as it allows toggling choices/states. I certainly agree with the suggested approach "copy answers from above" with the checkbox aligned right. Does this need to be on multiple tickets or on the last one?

Sure, will debug the other issues and also add support for other input types.

**Comment by raphaelm** [comment]
> On the UI, I felt a checkbox is a better control than a button as it allows toggling choices/states. 

Yes! I did not even think about it a checkbox instead of the button before your PR, but I really like it :)

> I certainly agree with the suggested approach "copy answers from above" with the checkbox aligned right. Does this need to be on multiple tickets or on the last one?

I think ideally it should be on all of them (except the first one, obviously) and if you activate it, it should copy the answers from the **first** ticket to the one the checkbox got activated on.

**Comment by chrisdamba** [comment]
Seems like I still have so much to learn from UX design :) It's actually starting to make sense if we remove the "unbind" which essentially renders the checkbox as a one-way action event and thus making way for a button! (the original design). Yeah, let's go for the button.

Still not sure why checkboxes and radios are not being picked, will have a look once again and let you know how that goes. Thanks for the feedback, really useful to me as it gradually points me in the right direction.

**Comment by raphaelm** [comment]
Feel free to ping me if you get totally stuck with the checkboxes and radios :) But some googling on jQuery and checkbox handling should bring up some useful results.

**Comment by chrisdamba** [comment]
@raphaelm Apologies for the slight delay in fix, been a bit snowed in with work lately but still very much into getting my first pull request changes done :)

**Comment by raphaelm** [comment]
No worries, we're not in a hurry!

**Comment by chrisdamba** [comment]
I am not sure what's causing the reduction in code coverage with the changes that I made in the PR. Can anyone help me to understand why that happened?

**Comment by raphaelm** [comment]
As you don't even touch the python code… please ignore it. I have to turn off the coveralls commits at some point, they grow to be annoying :(

**Comment by rixx** [comment]
It's neglegibly small; possibly based in your branch branching off from an earlier point of the master branch. Don't worry about it. :)

File: src/static/pretixpresale/js/ui/main.js [hunk 3]
```diff
@@ -32,14 +31,6 @@ $(function () {
     });
 });
 
-function display_copy() {
```
Comment (chrisdamba) [inline src/static/pretixpresale/js/ui/main.js]: This was redundant, forgot to take it out after the requested changes.

**Comment by chrisdamba** [comment]
Thanks @raphaelm ! :)

**Pull request merged**